tests/fixtures/models/simple_gaussian_mlp_model.py

```diff
@@ -7,7 +7,11 @@
 class SimpleGaussianMLPModel(Model):
     """Simple GaussianMLPModel for testing."""
 
-    def __init__(self, name, output_dim, *args, **kwargs):
+    def __init__(self,
+                 output_dim,
+                 *args,
+                 name='SimpleGaussianMLPModel',
```

Comment:
Sort arguments?

Reply:
Will fix it

Reply:
what about this one?

Reply:
Fixed. Not sure why it is not shown as Outdated.